packages/design-system/src/components/shortcut.tsx

```diff
@@ -9,6 +9,7 @@ const shortcutSymbolMap: Record<string, string> = {
   shift: "⇧",
   option: "⌥",
   backspace: "⌫",
+  click: "+click",
 };
```

Comment:
we might wanna switch to Sitnik's keyux for this later

Reply:
ofc not, edited the wrong file

Reply:
just realized I had forgotten there is kbd element, switched to using it, I am not sure how keyux will work with radix and stuff